pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -892,6 +897,148 @@ var _ = Describe("Snapshotter", func() {
 				})
 			})
 		})
+
+		Describe("Scenarios to update full snapshot lease", func() {
+			var (
+				ssr   *Snapshotter
+				lease *v1.Lease
+			)
+			BeforeEach(func() {
+				healthConfig.SnapshotLeaseRenewalEnabled = true
+				snapstoreConfig = &brtypes.SnapstoreConfig{Container: path.Join(outputDir, "default.bkp")}
+				store, err = snapstore.GetSnapstore(snapstoreConfig)
+				Expect(err).ShouldNot(HaveOccurred())
+
+				Expect(os.Setenv("POD_NAME", "test_pod")).To(Succeed())
+				Expect(os.Setenv("POD_NAMESPACE", "test_namespace")).To(Succeed())
+				os.Setenv("UNIT_TEST", "true")
```

Comment:
I removed this and then run the tests, they are working fine locally.
So, IMO you can try removing this from usage and it will work as long as snapshotter object has fakeK8sClient which you are updating it separately.
```
ssr.K8sClientset = fake.NewClientBuilder().Build()
```